openapi_python_client/parser/properties/model_property.py

```diff
@@ -19,6 +19,7 @@ class ModelProperty(Property):
     additional_properties: Union[bool, Property]
 
     template: ClassVar[str] = "model_property.py.jinja"
+    json_is_dict: ClassVar[bool] = True
```

Comment:
This should probably exist on the base `Property` as `False` right? I guess Jinja is handling it properly but I would expect an exception when checking `if not property.json_is_dict` for non- ModelProperty